src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -106,7 +104,12 @@ protected function handleFilter(
         }
 
         if (in_array($filterName, $availableFilterNames, true)) {
-            $searchNavigationRequest->addAttribute($filterName, $filterValue);
+            // This resolves the SW-451 issue about filter value conflict in storefront
```

Comment:
Upon click-testing I found out that the category filter does no longer work.
Clicking a category filter

1. Does not disable any other values.
1. Does not filter anything (and the URL is not changed at all).

There seems to be a console error, which is thrown on the initial load:

![image](https://user-images.githubusercontent.com/20555112/86106711-2dea3680-bac1-11ea-9b8d-89151dafbcda.png)